examples/methods/UpdateTaxes.yaml

```diff
@@ -0,0 +1,123 @@
+---
+rq:
+  GetBookingId:
+    summary: UpdateTaxes
+    value:
+      verb: UpdateTaxes
+      ota_property_id: lake_view_hotel
+      ota_property_password: very-secret-password
+      ota_property_sub_id: '10034818'
+      mya_property_id: 25678
+      guid: 6C08B96E-450D-4E6A-9933-7D0305730305
+      shared_secret : s3cr3ts4uc3
+      ota_cid: ota
+      taxes:
+        update:
```

Comment:
When I do `$ref: ../taxes.yaml` it renders to 
![ref_not_rendered](https://github.com/MyAllocator/build2us-apidocs/assets/114253347/2de0fe1a-19ab-4c62-a0a7-216e43d229b3)
But it starts working when I delete `$ref: ../taxes.yaml` from `CreateProperty` example.